global_costmap:
  global_frame: map
  robot_base_frame: base_footprint

  update_frequency: 10
  publish_frequency: 10
  transform_tolerance: 1

  static_map: true
 
